learing_journal/tests.py

```diff
@@ -83,7 +84,7 @@ def dummy_http_request_post(title, body, date, new_session):
 def test_lists_title(new_session):
     '''tests wether list() pull out correct data from db'''
     from .views.default import home
-    new_session.add(MyModel(title='test1', body='test2', date='test3'))
+    new_session.add(Journal(title='test1', body='test2', date=datetime.datetime.now()))
     new_session.flush()
     result = home(dummy_http_request(new_session))
```

Comment:
If it's a list view you should probably also test that one of the things returned in "result" is a list object containing the things you want, not just that there's a thing inside of whatever "result" is that has the value you want.